codec.proto

```diff
@@ -95,56 +94,47 @@ message ConsensusParams {
   VersionParams   version   = 4 [(gogoproto.nullable) = false];
 }
 
-message Data {
-  repeated bytes txs = 1;
-}
+message Data { repeated bytes txs = 1; }
 
 message Duration {
   int64 seconds = 1;
   int32 nanos   = 2;
 }
 
 message DuplicateVoteEvidence {
-  EventVote   vote_a              = 1;
-  EventVote   vote_b              = 2;
-  int64       total_voting_power  = 3;
-  int64       validator_power     = 4;
-  Timestamp   timestamp           = 5 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+  EventVote vote_a             = 1;
+  EventVote vote_b             = 2;
+  int64     total_voting_power = 3;
+  int64     validator_power    = 4;
+  Timestamp timestamp          = 5 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
 }
 
 message Event {
   string                  event_type = 1 [json_name = "type"];
-  repeated EventAttribute attributes = 2 [
-    (gogoproto.nullable) = false,
-    (gogoproto.jsontag)  = "attributes,omitempty"
-  ];
+  repeated EventAttribute attributes = 2 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "attributes,omitempty"];
 }
 
 message EventAttribute {
-  string  key   = 1;
-  string  value = 2;
-  bool    index = 3; // nondeterministic
+  string key   = 1;
+  string value = 2;
+  bool   index = 3; // nondeterministic
 }
 
 message EventBlock {
-  Block               block               = 1;
-  BlockID             block_id            = 2; // not present in v0.34.13
-  ResponseBeginBlock  result_begin_block  = 3;
-  ResponseEndBlock    result_end_block    = 4;
+  Block              block              = 1;
+  BlockID            block_id           = 2; // not present in v0.34.13
+  ResponseBeginBlock result_begin_block = 3;
+  ResponseEndBlock   result_end_block   = 4;
 }
 
-message EventTx {
-  TxResult tx_result = 1;
-}
+message EventTx { TxResult tx_result = 1; }
 
-message EventValidatorSetUpdates {
-  repeated Validator validator_updates = 1;
-}
+message EventValidatorSetUpdates { repeated Validator validator_updates = 1; }
 
 message EventVote {
   SignedMsgType event_vote_type   = 1 [json_name = "type"]; // should be present in the enum at the top of this file
-  uint64        height            = 2; // must be > 0
-  int32         round             = 3; // must be > 0
+  uint64        height            = 2;                      // must be > 0
+  int32         round             = 3;                      // must be > 0
```

Comment:
weird spacing here, I get that its to match up with the `event_vote_type` but it then doesn't line up with the `validator_address`